requirements/edx/github.txt

```diff
@@ -10,8 +10,8 @@
 # Third-party:
 -e git+https://github.com/cyberdelia/django-pipeline.git@1.5.3#egg=django-pipeline
 git+https://github.com/edx/django-wiki.git@ned/dj18#egg=django-wiki==0.0.4
--e git+https://github.com/edx/django-oauth2-provider.git@will/django-1.8-upgrade#egg=django-oauth2-provider==0.2.7-fork-edx-6
-git+https://github.com/edx/django-openid-auth.git@django1.8-upgrade#egg=django-openid-auth==0.8
+-e git+https://github.com/edx/django-oauth2-provider.git@0.2.7-fork-edx-7#egg=django-oauth2-provider==0.2.7-fork-edx-6
```

Comment:
I wonder if we shouldn't make the version match anyway. or use a tag like "0.2.7-fork-edx-6a", since we already horribly abusing the version syntax :)